crates/sui-types/src/sui_system_state.rs

```diff
@@ -324,6 +378,26 @@ impl SuiSystemState {
     }
 }
 
+fn create_narwhal_pubkey(bytes: &[u8]) -> Result<narwhal_crypto::PublicKey, FastCryptoError> {
+    narwhal_crypto::PublicKey::from_bytes(bytes)
+}
+
+fn create_narwhal_net_pubkey(
+    bytes: &[u8],
+) -> Result<narwhal_crypto::NetworkPublicKey, FastCryptoError> {
+    narwhal_crypto::NetworkPublicKey::from_bytes(bytes)
+}
+
+fn create_multiaddr(bytes: Vec<u8>) -> Result<Multiaddr, multiaddr::Error> {
+    Multiaddr::try_from(bytes)
+}
+
+pub fn create_authority_pubkey_bytes(
```

Comment:
Do you still need these?

Reply:
> Do you still need these?

3 out of 4 hey are used in the verification function itself (the 4th one's use is a bit weird - I think we should be able to verify metadata and return `metadata.pubkey_bytes` but I am not sure) . Technically only 2 out of these 3 are called more than once  but it looks more consistent to me to have them all as helpers. Will move them closer to their respective uses, though, to make their utility more clear.

Reply:
We had them before to better share code. But now I don't see the benefit of having extra functions that just makes one function call. It doesn't really make the code cleaner/less in anyway. Could we just inline the call and get rid of them?

Reply:
Fair. Done!